### PR TITLE
Add support for mongo v3.2 $unwind

### PR DESCRIPTION
**Pull request by rintcius**
Depends on #3375
  
 #qz-3574 done

### CODE REVIEW

File: mongodb/src/main/scala/quasar/physical/mongodb/workflow/WorkflowOpCore.scala [hunk 1]
```diff
@@ -917,7 +927,13 @@ object WorkflowOpCoreF {
             Nil)
         case $LimitF(_, count)  => Terminal("$LimitF" :: wfType, Some(count.shows))
         case $SkipF(_, count)   => Terminal("$SkipF" :: wfType, Some(count.shows))
-        case $UnwindF(_, field) => Terminal("$UnwindF" :: wfType, Some(field.shows))
+        case $UnwindF(_, field, includeArrayIndex, preserveNullAndEmptyArrays) =>
```
Comment (jsantos17) [inline mongodb/src/main/scala/quasar/physical/mongodb/workflow/WorkflowOpCore.scala]: Can we add a case for the common case where both `includeArrayIndex` and `preserveNullAndEmptyArrays` are `None`? Might help keep the noise down in `.txt` expectations.

**Comment by rintcius** [comment]
@jsantos17 updated

**Comment by jsantos17** [comment]
The failure on Mimir looks legitimate. Any idea what's going on there?

**Comment by alissapajer** [comment]
The failures in mimir and ML are present on master :(

**Comment by rintcius** [comment]
Closing, merged as part of #3412

**Pull request closed without merging**